File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/ApplicationClass.java
package com.atomykcoder.atomykplay;

import android.app.Application;
import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.graphics.Color;
import android.os.Build;
import android.widget.Toast;

import androidx.core.splashscreen.SplashScreen;

public class ApplicationClass extends Application {

    public static final String CHANNEL_ID = "MUSIC_NOTIFICATION";

    @Override
    public void onCreate() {
        super.onCreate();
        createNotificationChannel();
    }

    private void createNotificationChannel() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            NotificationChannel notificationChannel = new NotificationChannel(CHANNEL_ID, "Music Notification", NotificationManager.IMPORTANCE_LOW);
            notificationChannel.setDescription("Music playback notification are controlled from here");
            notificationChannel.setImportance(NotificationManager.IMPORTANCE_LOW);
            notificationChannel.setBypassDnd(false);
            notificationChannel.enableLights(false);
            notificationChannel.enableVibration(false);
            notificationChannel.setShowBadge(false);

            NotificationManager manager = (NotificationManager) getSystemService(NOTIFICATION_SERVICE);
            manager.createNotificationChannel(notificationChannel);

        }

    }

    public void showToast(String s) {
        Toast.makeText(getApplicationContext(), s, Toast.LENGTH_SHORT).show();
    }
}

File: AtomykPlay/app/src/main/java/com/atomykcoder/atomykplay/ApplicationClass.kt
package com.atomykcoder.atomykplay

import android.app.Application
import android.app.NotificationChannel
import android.app.NotificationManager
import android.os.Build

class ApplicationClass : Application() {
    init {
        instance = this
    }

    override fun onCreate() {
        super.onCreate()
        createNotificationChannel()
    }

    private fun createNotificationChannel() {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            val notificationChannel = NotificationChannel(
                CHANNEL_ID,
                "Music Notification",
                NotificationManager.IMPORTANCE_LOW
            )
            notificationChannel.description = "Music playback notification are controlled from here"
            notificationChannel.importance = NotificationManager.IMPORTANCE_LOW
            notificationChannel.setBypassDnd(false)
            notificationChannel.enableLights(false)
            notificationChannel.enableVibration(false)
            notificationChannel.setShowBadge(false)
            val manager = getSystemService(NOTIFICATION_SERVICE) as NotificationManager
            manager.createNotificationChannel(notificationChannel)
        }
    }

    companion object {
        const val CHANNEL_ID = "MUSIC_NOTIFICATION"
        lateinit var instance: ApplicationClass
            private set
    }
}